### PR TITLE
simple test sends the first time success notification.

### PR DESCRIPTION
**Pull request by bingzhang**
## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have signed the CLA
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by bingzhang** [comment]
@robkooper if we check the date, we still need to calculate the # failures or # successes. This also depends on how many times the script run per day. I think it is similar to what the current pull request does.

**Comment by robkooper** [comment]
I was thinking something along, this will send a message on success after failure, and the first time after midnight.

```
if [ job fails ]; then
  send failure message
  remove memory file
else
  today = $(date +'%Y-%m-%d') 
  if [ ! -e memoryfile ]; then
     send success message
  else if [ "$(cat memoryfile)" != "$today"]; then
     send success message
  fi
  echo $today > memoryfile
fi
```

File: scripts/tester/tester.sh [hunk 1]
```diff
@@ -83,15 +84,24 @@ while [ $FILE_EXTRACTED -eq 0 ]; do
...
 echo "File extraction complete."
-    
+
+today=$(date +'%Y-%m-%d')
+if [ ! -e memoryfile ]; then
+     post_message "Upload+extract test script success!"
+elif [ "$(cat memoryfile)" != "$today" ]; then
+     post_message "Upload+extract test script success!"
+fi
+
```
Comment (robkooper) [inline scripts/tester/tester.sh:101]: I think you need to write the $today to `memoryfile`

Comment (lmarini) [reply scripts/tester/tester.sh]: @robkooper is the fix from @bingzhang good? Can we merge this?

**Pull request merged**